src/main/webapp/script.js

```diff
@@ -58,4 +59,32 @@ async function displayDailyDeed() {
 
     deedTitle.innerText = daily_deed.title;
     deedDescription.innerText = daily_deed.description;
+}
+
+/**
+ *displays the current date using the built in Date() class
+ */
+function displayDate() {
+    const current_date = document.getElementById("curDate");
+
+    var months = new Object();
+    months[1] = "January";
```

Comment:
Can this not start at months[0] ? Seems a bit weird.

Reply:
Yes that sounds good! I was treating months sort of like a hash table so the 1 was more of like a key, but I can start these at 0. It'll actually complement the Date.getMonth() function better since it starts counting the months from zero.